public/javascripts/data.js

```diff
@@ -346,7 +346,7 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
 
             // create binding based on kind
             var kind = control.kind.toLowerCase();
-            if (kind == 'device id') 
```

Comment:
Sorry, that's me editor deleting trailing whitespace on file save. Hope that's OK to submit.

Reply:
if this were an active project i'd grouse but as is it's totally fine.